modules/sdk-coin-bsc/test/unit/transactionBuilder/send.ts

```diff
@@ -0,0 +1,160 @@
+import { getBuilder } from '../getBuilder';
+import { Ecdsa, TransactionType } from '@bitgo/sdk-core';
+import { KeyCombined } from 'modules/sdk-core/src/account-lib/mpc/tss/ecdsa/types';
+
+describe('BSC Transfer Builder', () => {
+  describe('TSS Signing with 2 of 3', function () {
+    let MPC: Ecdsa;
+    let A_combine: KeyCombined, B_combine: KeyCombined, C_combine: KeyCombined;
+    let serializedTransaction: Buffer;
+
+    before('key generation and unsigned tx building', async () => {
+      MPC = new Ecdsa();
+      const A = await MPC.keyShare(1, 2, 3);
+      const B = await MPC.keyShare(2, 2, 3);
+      const C = await MPC.keyShare(3, 2, 3);
+
+      A_combine = MPC.keyCombine(A.pShare, [B.nShares[1], C.nShares[1]]);
+      B_combine = MPC.keyCombine(B.pShare, [A.nShares[2], C.nShares[2]]);
+      C_combine = MPC.keyCombine(C.pShare, [A.nShares[3], B.nShares[3]]);
+
+      const txBuilder = getBuilder('tbsc');
+      txBuilder.type(TransactionType.SingleSigSend);
+      txBuilder.value('1000');
+      txBuilder.contract('0xA2959D3F95eAe5dC7D70144Ce1b73b403b7EB6E0');
+      txBuilder.counter(1);
+      txBuilder.fee({
+        fee: '20',
+        gasLimit: '30',
+      });
+      const unsignedTransaction = await txBuilder.build();
+      serializedTransaction = Buffer.from(unsignedTransaction.toBroadcastFormat());
+    });
+
+    it('A and B signing', () => {
+      const B_sign_share = MPC.signShare(B_combine.xShare, B_combine.yShares[1]);
+
+      const A_sign_convert = MPC.signConvert({
+        xShare: A_combine.xShare,
+        yShare: A_combine.yShares[2],
+        kShare: B_sign_share.kShare,
+      });
+
+      const B_sign_convert = MPC.signConvert({
+        wShare: B_sign_share.wShare,
+        aShare: A_sign_convert.aShare,
+      });
+
+      const A_resign_convert = MPC.signConvert({
+        bShare: A_sign_convert.bShare,
+        muShare: B_sign_convert.muShare,
+      });
+
+      const B_sign_combine = MPC.signCombine({
+        gShare: A_resign_convert.gShare!,
+        signIndex: {
+          i: 1,
+          j: 2,
+        },
+      });
+
+      const A_sign_combine = MPC.signCombine({
+        gShare: B_sign_convert.gShare!,
+        signIndex: {
+          i: 2,
+          j: 1,
+        },
+      });
+
+      const A_sign = MPC.sign(serializedTransaction, A_sign_combine.oShare, B_sign_combine.dShare);
+      const B_sign = MPC.sign(serializedTransaction, B_sign_combine.oShare, A_sign_combine.dShare);
+      const signature = MPC.constructSignature([A_sign, B_sign]);
+
+      MPC.verify(serializedTransaction, signature).should.be.true;
+    });
+
+    it('A and C signing', () => {
```

Comment:
Are tests for all three signing pairs needed? Should I just have a test for one pair instead?